models/zeroex/base/zeroex_base_api_fills_deduped.sql

```diff
@@ -0,0 +1,102 @@
+{{  config(
+        tags=['dunesql'],
+        alias = alias('api_fills_base_deduped'),
+        materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
same idea here

Reply:
while you wait for decode table @RantumBits -- can you update this partition to `block_month` too?